style.css

```
@@ -0,0 +1,119 @@
+* {
+    box-sizing: border-box;
+    font-family: 'IBM Plex Sans', sans-serif;
+}
+/* 
+html,
+body{
+    overflow: hidden;
+    width: 100%;
+    height: 100%;
+} */
+
+body {
+    margin: 0;
+    background: rgb(116,255,0);
+    background: linear-gradient(0deg, rgba(116,255,0,1) 0%, rgba(255,255,255,1) 100%) fixed;
```

1. Really appreciate that you include a fallback color on line 15 👍🏻
1. For your gradient, you've actually written a little bit more code than is necessary; here's a simplified version:
```suggestion
    background: linear-gradient(rgb(255,255,255), rgb(116,255,0)) fixed;
```

Things I've changed:
- Since your colors are solid, I switched from `rgba` to `rgb` and removed the final value (`1` in both cases). You only need `rgba` when you're adding a value for [opacity](https://mica-web.github.io/learn/css/opacity).
- The default for a linear gradient is to run from the top to the bottom of the page. So, by switching the order you reference your colors (starting with white, followed by the bright green), I was able to eliminate both the degrees and the percentages from this declaration.